tests/nnstreamer_plugins/unittest_plugins.cpp

```diff
@@ -2849,6 +2849,75 @@ TEST (test_tensor_transform, orc_performance)
 }
 #endif /* HAVE_ORC */
 
+#ifdef ENABLE_TENSORFLOW_LITE
+#define wait_for_element_state(element,state) do { \
+  GstState cur_state = GST_STATE_VOID_PENDING; \
+  GstStateChangeReturn ret; \
+  gint counter = 0;\
+  ret = gst_element_set_state (element, state); \
+  EXPECT_TRUE (ret != GST_STATE_CHANGE_FAILURE); \
+  while (cur_state != state && counter < 20) { \
+    g_usleep (50000); \
+    counter++; \
+    ret = gst_element_get_state (element, &cur_state, NULL, 5 * GST_MSECOND); \
+    EXPECT_TRUE (ret != GST_STATE_CHANGE_FAILURE); \
+  } \
+  EXPECT_TRUE (cur_state == state); \
+  g_usleep (50000); \
+} while (0)
```

Comment:
This is worth to be a static-inline function...?

Reply:
I wanted to surely guarantee the pipeline state changes (playing > pause > update properties and set playing again)